port/zos390/omrvmem.c

```diff
@@ -579,7 +581,7 @@ reservePagesAboveBar(struct OMRPortLibrary *portLibrary, J9PortVmemIdentifier *i
 
 		Trc_PRT_vmem_reservePagesAboveBar_allocate_4K_pages_in_2to32G_area(numSegments);
 		allocator = OMRPORT_VMEM_RESERVE_USED_J9ALLOCATE_4K_PAGES_IN_2TO32G_AREA;
-		ptr = omrallocate_4K_pages_in_userExtendedPrivateArea(numSegments, userExtendedPrivateAreaMemoryType, ttkn);
+		ptr = omrallocate_4K_pages_in_userExtendedPrivateArea(numSegments, userExtendedPrivateAreaMemoryType, ttkn, startAddress);
```

Comment:
I guess debug prints have some value so would be better to update them with extra `startAddress` parameter to keep up to date

Reply:
**startAddress** is added to a trace line in b333ae71e85896febf6ce8260d2d28493524dddf